src/main/java/com/redhat/devtools/intellij/commonUiTestLibrary/fixtures/dialogs/projectManipulation/pages/JavaProjectSecondPage.java

```diff
@@ -0,0 +1,59 @@
+/*******************************************************************************
+ * Copyright (c) 2020 Red Hat, Inc.
+ * Distributed under license by Red Hat, Inc. All rights reserved.
+ * This program is made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution,
+ * and is available at https://www.eclipse.org/legal/epl-v20.html
+ *
+ * Contributors:
+ * Red Hat, Inc. - initial API and implementation
+ ******************************************************************************/
+package com.redhat.devtools.intellij.commonUiTestLibrary.fixtures.dialogs.projectManipulation.pages;
+
+import com.intellij.remoterobot.RemoteRobot;
+import com.intellij.remoterobot.data.RemoteComponent;
+import com.intellij.remoterobot.fixtures.DefaultXpath;
+import com.intellij.remoterobot.fixtures.FixtureName;
+import com.intellij.remoterobot.fixtures.JCheckboxFixture;
+import com.redhat.devtools.intellij.commonUiTestLibrary.fixtures.dialogs.projectManipulation.pages.abstractPages.AbstractNonterminalPage;
+import org.jetbrains.annotations.NotNull;
+
+/**
+ * New Project dialog java project second page fixture
+ *
+ * @author zcervink@redhat.com
+ */
+@DefaultXpath(by = "MyDialog type", xpath = "//div[@class='DialogRootPane']")
+@FixtureName(name = "New Project Dialog")
+public class JavaProjectSecondPage extends AbstractNonterminalPage {
+    public JavaProjectSecondPage(@NotNull RemoteRobot remoteRobot, @NotNull RemoteComponent remoteComponent) {
+        super(remoteRobot, remoteComponent);
+    }
+
+    /**
+     * Check the 'Create project from template' checkbox
+     */
+    public void selectCreateProjectFromTemplateCheckBox() {
+        createNewProjectFromTemplateCheckBox().select();
+    }
+
+    /**
+     * Uncheck the 'Create project from template' checkbox
+     */
+    public void unselectCreateProjectFromTemplateCheckBox() {
```

Comment:
Too long method names. Please use shorter names and omit object suffixes that are aligned to the method. Ie. uinstead of 
```
selectCreateProjectFromTemplateCheckBox
```
you can use 
```
toggleCreateProjectFromTemplate(boolean toggle);
```
And then you can remove also unselectCreateProjectFromTemplateCheckBox.

Reply:
Fixed and reworked.